docker/RDMA/Dockerfile

```diff
@@ -17,14 +17,14 @@
 FROM apache/incubator-crail:1.1
 MAINTAINER Apache Crail <dev@crail.apache.org>
 
-# TODO: automate update version
-ARG DISNI_COMMIT="v1.7"
-
 RUN echo "Crail-$LOG_COMMIT install rdma libraries and autotools" && \
```

Comment:
Can we remove LOG_COMMIT? It seems not to be initialized?

Reply:
Good question. Let me check where this comes from.

Reply:
Should be initialized by base image (ENV).

Reply:
Ah good, then it is fine.

Reply:
Thx. Will merge.